ts/components/leftPane/LeftPaneSearchHelper.tsx

```diff
@@ -298,10 +306,29 @@ export class LeftPaneSearchHelper extends LeftPaneHelper<LeftPaneSearchPropsType
     );
   }
 
-  // This is currently unimplemented. See DESKTOP-1170.
   getConversationAndMessageAtIndex(
     _conversationIndex: number
   ): undefined | { conversationId: string; messageId?: string } {
+    if (_conversationIndex < 0) {
+      return undefined;
+    }
+    let pointer = _conversationIndex;
+    for (const list of this.allResults()) {
+      if (list.isLoading) {
+        continue;
+      }
+      if (pointer < list.results.length) {
+        const result = list.results[pointer];
+        return 'conversationId' in result
+          ? // it is a message result
+            {
+              conversationId: result.conversationId,
+              messageId: result.id,
+            }
+          : { conversationId: result.id };
+      }
+      pointer -= list.results.length;
+    }
```

Comment:
nit pick but can we also clear the search when you do ctrl+N?

Reply:
Fixed, see LeftPane.tsx code

Reply:
Or did you mean "new conversation" shortcut `ctrl+n`? I guess compose helper should also get simillar functionality to keep UX consistent